src/util/crypto.ts

```diff
@@ -79,3 +79,16 @@ export const verifySignedMessage = (
     throw new Error('Failed to open signed message with given public key')
   return JSON.parse(encodeUTF8(openedMessage))
 }
+
+/**
+ * Helper method to check  if all the fields given are wtihin the filenames
```

Comment:
```suggestion
 * Helper method to check if all the field IDs given are within the filenames
```

Reply:
it's time for me to install a spell-checker @__@